src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -4608,6 +4617,28 @@ public static Value retrieve_item(ScriptRuntime controller, final Value item) {
     return retrieve_item(controller, DataTypes.ONE_VALUE, item);
   }
 
+  public static Value retrieve_price(ScriptRuntime controller, final Value arg1, final Value arg2) {
+    int arg1Value = (int) arg1.intValue();
+    int arg2Value = (int) arg2.intValue();
+
+    boolean countThenItem =
+        arg1.getType().equals(DataTypes.INT_TYPE) || arg1.getType().equals(DataTypes.FLOAT_TYPE);
+
+    int count = countThenItem ? arg1Value : arg2Value;
+    int item = countThenItem ? arg2Value : arg1Value;
+
+    if (count <= 0) {
+      return RuntimeLibrary.continueValue();
```

Comment:
This can't be right. This method returns an int, but the continueValue is a boolean.

Reply:
oh my bad I copied that from retrieve_item, it should return 0 or -1 I'll fix it.